include/SmurfBuilder.h

```diff
@@ -35,6 +35,21 @@ class SmurfBuilder : public G3EventBuilder{
     void       setDebug(bool d) { debug_ = d;    };
     const bool getDebug()       { return debug_; };
 
+    void setEncode(bool b) { encode_timestreams_ = b; };
+    const bool getEncode() {return encode_timestreams_;};
```

Comment:
const should go on the right, not the left.
https://isocpp.org/wiki/faq/const-correctness#const-member-fns